common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -587,13 +733,32 @@ def test_can_show_answer(self):
         self.problem_page.wait_for_show_answer_notification()
 
 
+class CheckboxProblemTypeTestNonRandomized(CheckboxProblemTypeBase, NonRandomizedProblemTypeTestMixin):
+    """
+    Tests for the non-randomized checkbox problem
+    """
+
+    def get_problem(self):
```

Comment:
We are using similar code in multiple places with a few changes in arguments. What about moving this into some base class or mixin so we can inherit the basic implementation of this method in all places?